### PR TITLE
Refactor path for configuration file, option(s) and Docker image

### PR DESCRIPTION
**Pull request by gab-arrobo**

### CODE REVIEW

**Comment by thakurajayL** [comment]
As such looks good.. Will this break any other work for Canonical?

**Comment by gab-arrobo** [comment]
> As such looks good.. Will this break any other work for Canonical?

I do not think this should break anything. However, I can double check with them. The changes here are consistent with the other NFs. So, if these changes break something on the Canonical side, the changes in all of the other NFs will have similar effect

File: configmodels/model_slice.go [hunk 1]
```diff
@@ -15,7 +15,7 @@
 package configmodels
 
 type Slice struct {
-	SliceName string `json:slice-name,omitempty"`
+	SliceName string `json:"slice-name,omitempty"`
```
Comment (gab-arrobo) [inline configmodels/model_slice.go:18]: @thakurajayL, 
FYI, the tests fail because of this change

Comment (gab-arrobo) [reply configmodels/model_slice.go:18]: This is fixed now

**Comment by gab-arrobo** [comment]
@patriciareinoso, any objection about this PR?

**Comment by patriciareinoso** [comment]
We are currently using `/bin/webconsole --webuicfg <path>` to run the service (same for the other NFs),  so it will break our stuff. However it's only a small change to take care of when updating the images.

**Comment by thakurajayL** [comment]
@patriciareinoso its not just webconsole but also other NFs have gone through similar change.

**Review by thakurajayL — APPROVED** [review]

**Pull request merged**